modules/host/contractmanager/consts.go

```diff
@@ -42,7 +42,7 @@ const (
 	// folderAllocationStepSize is the amount of data that gets allocated at a
 	// time when writing out the sparse sector file during a storageFolderAdd or
 	// a storageFolderGrow.
-	folderAllocationStepSize = 50e6
+	folderAllocationStepSize = 1 << 35
```

Comment:
seems like quite a big jump, from 50MB to 30GB. Will this be a problem for people without `Truncate`?